sdk/python/kubeflow/training/api/training_client.py

```diff
@@ -81,6 +89,109 @@ def __init__(
             )
         self.job_kind = job_kind
 
+    def train(
+        self,
+        name: str = None,
+        namespace: str = None,
+        num_workers: int = 1,
+        num_procs_per_worker: int = 1,
+        pvc: Dict[Literal["name", "claimName"], str] = None,
```

Comment:
I am not sure if Data Scientists should be worry about PVC creation while using this API.
Should we create this PVC on a fly and default storage class will be responsible to assign appropriate volume for it ?
WDYT @johnugeorge @deepanker13 ?

Reply:
yes @andreyvelich , discussed it with Johnu, will take a storage config instead.